developer-guides/deeplink/README.md

```diff
@@ -0,0 +1,43 @@
+# Deeplink
+
+**This feature is only supported in our iOS native app right now.**
+
+## General information
+
+> rocketchat://
+
+## Authentication
+
+### Without token
+
+**Params:**
+
+- host: The host of the server;
+
+**Example:**
+
+> rocketchat://auth?host=foo.bar.com
+
+### With token
+
+**Params:**
+
+- host: The host of the server;
+- token: The token of the user to be authenticated;
+- userId: The id of the user to be authenticated;
```

Comment:
Mark token and userId as optional. If not provided, we connect to the server, but show login form/methods.

Reply:
@cardoso I added two options (with token and without token) but let's make only one, simpler :-)